frontend/src/components/elements/Audio/Audio.tsx

```diff
@@ -15,48 +15,35 @@
  * limitations under the License.
  */
 
-import React, { PureComponent, ReactNode } from "react"
+import React, { ReactElement, useRef } from "react"
 import { Map as ImmutableMap } from "immutable"
 import { buildMediaUri } from "lib/UriUtil"
 
-export interface Props {
+export interface AudioProps {
   width: number
   element: ImmutableMap<string, any>
 }
 
-class Audio extends PureComponent<Props> {
-  private audioRef = React.createRef<HTMLAudioElement>()
-
-  public componentDidMount = (): void => {
-    this.updateTime()
-  }
-
-  public componentDidUpdate = (): void => {
-    this.updateTime()
-  }
-
-  private updateTime(): void {
-    if (this.audioRef.current) {
-      const { element } = this.props
-
-      this.audioRef.current.currentTime = element.get("startTime")
+export default function Audio({ element, width }: AudioProps): ReactElement {
+  const audioRef = useRef<HTMLAudioElement>(null)
+  const updateTime = (): void => {
+    if (audioRef.current) {
+      audioRef.current.currentTime = element.get("startTime")
     }
   }
-
-  public render(): ReactNode {
-    const { element, width } = this.props
-
-    const uri = buildMediaUri(element.get("url"))
-    return (
-      <audio
-        ref={this.audioRef}
-        controls
-        src={uri}
-        className="stAudio"
-        style={{ width }}
-      />
-    )
-  }
+  React.useEffect(() => {
+    updateTime()
```

Comment:
At this point, I'd probably take the content of the `updateTime` function and put it here since it's now the only place it's used.

Reply:
yes. will get do it inline